README.md

```diff
@@ -31,3 +31,237 @@ A dashboard showing the tests running against browser engines can be seen at [wp
 ## Building "locally"
 
 For quick local iteration, run `make`; this will use a web service to build the standard, so that you don't have to install anything. See more in the [Contributor Guidelines](https://github.com/whatwg/meta/blob/main/CONTRIBUTING.md#building).
+
+## Formatting
```

Comment:
Pretty sure this repository doesn't use Anne formatting rules, but instead uses much more Markdown, and does include breaks inside.